pipit/readers/nsight_sqlite_reader.py

```diff
@@ -284,15 +288,12 @@ def read(self) -> pipit.trace.Trace:
                 how="inner",
             )
         )
-        # TODO: can get rid of the apply if we use an Arrow ListDtype for children
-        # globally
-        children = calls_that_launch["index_y"].apply(lambda x: [x])
         # Convert to numpy otherwise the index messes stuff up
-        trace_df.loc[calls_that_launch["index_x"].to_numpy(), "_children"] = (
-            children.to_numpy()
+        trace_df.loc[calls_that_launch["index_x"].to_numpy(), "_kernel_launch"] = (
+            calls_that_launch["index_y"].to_numpy()
```

Comment:
If I'm reading this right, the trace_df returned here will no longer have a column called `_children`. Would this be a problem for functions that consume this trace, namely `filter_event_name`? I added a comment in that function to indicate where `_children` is used.

Reply:
The comment on line 278 above also mentions `_children`, that comment should be updated to reflect what's happening here. If both `_children` and `_kernel_launch` are going to be in the `trace_df` then that comment should mention what each contains.